packages/plugins/model-define/detectron-fasterrcnn-model-define/package.json

```diff
@@ -26,5 +26,17 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "pipcook": {
+    "category": "modelDefine",
+    "datatype": "image",
+    "method": "detect"
+  },
+  "conda": {
+    "python": "3.7",
+    "dependencies": {
+      "torch": "*",
+      "torchvision": "*"
```

Comment:
does this support pip install from git. For example, to install detectron, we need to run `pip install 'git+https://github.com/facebookresearch/detectron2.git'`

Reply:
Will support :)

Reply:
Supported at https://github.com/alibaba/pipcook/commit/775b1c496f0b658f50152f32f98fce3fcd12bd04#diff-32df0838fb24296b5d38739a7cf1b1fdR40 :)